src/components/diagrams/diagram-common.ts

```diff
@@ -296,7 +296,7 @@ export const useDiagram = () => {
 
 export interface Svg {
     svg: string | null;
-    metadata: SLDMetadata | null;
+    metadata: SLDMetadata | string | null;
```

Comment:
may be you can add a comment on why the second type is a string instead of DiagramMetadata ? 
fix in the backend to be done later ?

Reply:
yes